uint32 id
float32 Long
float32 Lat
float32 velo_Long
float32 velo_Lat
uint32 dyn_prop
float32 rcs
